examples/gtc_final_report.py

```diff
@@ -18,7 +18,7 @@
     sys.exit(-1)
 
 try:
-    manifest = BeadPoolManifest(args.manifest)
+    manifest = BeadPoolManifest.BeadPoolManifest(args.manifest)
```

Comment:
Does the reference to this import need to change?

Reply:
Reference should be fine, checked with running gtf_final_report_matrix.py. There is some ambiguity as the module name and class name are the same.